relational_operators/TextScanOperator.cpp

```diff
@@ -234,11 +234,10 @@ void TextScanOperator::receiveFeedbackMessage(const WorkOrder::FeedbackMessage &
   }
 }
 
-
```

Comment:
I am not aware of this convention. It seems that many other relational operators don't follow it either. Did we document it somewhere?